indexer/services/comlink/src/controllers/api/v4/vault-controller.ts

```diff
@@ -294,21 +295,22 @@ router.get(
 
 async function getVaultSubaccountPnlTicks(
   vaultSubaccountIds: string[],
-  resolution?: PnlTickInterval,
+  resolution: PnlTickInterval,
 ): Promise<PnlTicksFromDatabase[]> {
   if (vaultSubaccountIds.length === 0) {
     return [];
   }
-  let pnlTickInterval: PnlTickInterval;
-  if (resolution === undefined) {
-    pnlTickInterval = PnlTickInterval.day;
+
+  let windowSeconds: number;
+  if (resolution === PnlTickInterval.day) {
```

Comment:
nit: maybe define a function that converts from resolution to seconds? thinking about the possibility of adding more resolution values in the future

Reply:
This doesn't work with the underlying SQL query.